DESCRIPTION

```diff
@@ -32,8 +32,8 @@ Collate:
   'print.R'
   'internal.R'
 Remotes: plotly/dash-html-components@55c3884,
-  plotly/dash-core-components@c107e0f,
-  plotly/dash-table@3058bd5
+  plotly/dash-core-components@7fa7feb,
```

Comment:
I think this needs to point here instead
https://github.com/plotly/dash-core-components/commit/fc153b4e8b980231be3d49c0f20281b2fc9678cf